zerver/openapi/markdown_extension.py

```diff
@@ -76,6 +76,28 @@
     "string": "demo",
     "boolean": False,
 }
+CURL_EXCLUDE_DICTIONARY: Dict[str, List[str]] = {
+    "/messages/{message_id}/reactions:post": ["emoji_code", "reaction_type"],
+    "/messages:get": ["client_gravatar", "apply_markdown", "use_first_unread_anchor"],
+    "/users/me/subscriptions/muted_topics:patch": ["stream_id"],
+    "/messages/{message_id}/reactions:delete": ["emoji_code", "reaction_type"],
+    "/typing:post": ["topic"],
+    "/messages/{message_id}:patch": ["stream_id"],
+    "/user_groups/{user_group_id}/members:post": ["delete"],
+}
+CURL_INCLUDE_DICTIONARY: Dict[str, List[str]] = {
+    "/users/me/subscriptions:post": ["subscriptions"],
+    "/streams:get": [""],
+    "/events:get": ["queue_id", "last_event_id"],
+    "/streams/{stream_id}:patch": ["new_name", "description", "is_private"],
+    "/settings/display:patch": ["left_side_userlist", "emojiset"],
+    "/settings/notifications:patch": [
+        "enable_offline_push_notifications",
+        "enable_online_push_notifications",
+    ],
+    "/users/me/subscriptions:delete": ["subscriptions"],
+    "/register:post": ["event_types"],
+}
```

Comment:
I feel like long-term we may want to migrate to an `x-curl-example-parameters` property that has `include/exclude` subproperties, in the `zulip.yaml` file.  And perhaps we'd be able to define the multiple-example version with a `oneOf` list, with the second one having a `description_heading` field or something.